.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -149,6 +149,27 @@ jobs:
       - run: gcloud spanner instances create test-instance --config=emulator-config --description="Test Instance" --nodes=1
 
       # run tests
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - name: Start Local Server
+        run: go run github.com/GoogleCloudPlatform/spanner-migration-tool web &
```

Comment:
What version of SMT will this use? Not sure if this will pull SMT from Golang repo and execute it or use the version of SMT that is part of this PR. 

We want to run the version of SMT that is updated as part of this PR.